### PR TITLE
Diona Nymphs Fixes + Ear Slot

### PR DESCRIPTION
**Pull request by Ordosian**
## What Does This PR Do
Diona Nymphs were unable to resist out of their holder state. This is now fixed. Nymphs and other mobs with holders (PAI, Drones) can now be put in the Ear Slot. Nymphs have a new in-hand, taken from Aurora Station.

## Why It's Good For The Game
Nymphs can now resist out of their holder states and can be worn in the ear slot along with most other holder mobs.

## Testing
Booted up a server, spawned in human and nymphs, and tested resisting out of the holder states and visual tests.

## Changelog
:cl:
add: Nymphs and other holder mobs can be worn in the ear slot.
tweak: Tweaked the sprite for Nymphs when picked up. Sprite from AuroraStation
fix: Fixed Nymphs not being able to resist when picked up.
/:cl:

### CODE REVIEW

File: paradise.dme [hunk 1]
```diff
@@ -13,7 +13,6 @@
 #include "_maps\__MAP_DEFINES.dm"
 #include "_maps\base_map.dm"
 #include "_maps\map_datums.dm"
-#include "code\__byond_version_compat.dm"
```
Comment (GDNgit) [inline paradise.dme:16]: Update your dme

File: code/modules/mob/living/simple_animal/friendly/diona_nymph.dm [hunk 2]
```diff
@@ -96,7 +96,10 @@
 		..()
 
 /mob/living/simple_animal/diona/run_resist()
-	split()
+	if(isdiona(loc))
+		split()
```
Comment (GDNgit) [inline code/modules/mob/living/simple_animal/friendly/diona_nymph.dm:100]: just do if(!split) return ..()

**Comment by Ordosian** [comment]
Fixing the DME. Please wait. Its being a bit.. annoying

**Comment by Ordosian** [comment]
I can add some of the PAI and Mice sprites to go on the ear. Ones that fit. If that is requested.

**Comment by Ordosian** [comment]
Added some PAI sprites and other holder mobs such as mice that fit the shoulder aesthetic. Diona Nymph inhand sprite has been changed.

**Review by Sirryan2002 — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Pull request merged**